.github/actions/build_documentation/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6
+
+COPY documentation /documentation
+
+RUN apt-get update \
+ && apt-get install -yqq curl \
+ && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
```

Comment:
Any reason to use an older version of node?

Reply:
older version? This install 10.x, which is recommended I believe. the `8`, in `setup_8`  references the google v8 engine its built on, no?

Reply:
The comment at the top of that script says it's Node.js 8.x. v12 is current LTS: https://www.google.com/search?client=safari&rls=en&q=current+nodejs+version+lts&ie=UTF-8&oe=UTF-8

I noticed this when I was building the action for test running. Not a big deal, but might as well be current.